go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/GeertJohan/go.rice v1.0.0
```

Comment:
At line 1, change to `module github.com/KyberNetwork/solana-go`

Reply:
I think this is not a good idea because we have to replace all `import "github.com/gagliardetto/solana-go" with "github.com/KyberNetwork/solana-go".

You can see other forks also don't do this https://github.com/Blockdaemon/solana-go/blob/main/go.mod and https://github.com/certusone/solana-go/blob/main/go.mod.

Reply:
But when I install our lib, it cause error as follow

>  go get github.com/KyberNetwork/solana-go@ws_reconnect 
go: downloading github.com/KyberNetwork/solana-go v0.0.1-0.20221117100728-0c161db56677
go: github.com/KyberNetwork/solana-go@v0.0.1-0.20221117100728-0c161db56677: parsing go.mod:
        module declares its path as: github.com/gagliardetto/solana-go
                but was required as: github.com/KyberNetwork/solana-go

Reply:
There is a solution here https://stackoverflow.com/questions/14323872/using-forked-package-import-in-go